Lib/sqlite3/dump.py

```diff
@@ -7,6 +7,14 @@
 # future enhancements, you should normally quote any identifier that
 # is an English language word, even if you do not have to."
 
+def quote_name(name):
+    return '"{0}"'.format(name.replace('"', '""'))
+
+
+def quote_value(value):
+    return "'{0}'".format(value.replace("'", "''"))
```

Comment:
I wasn't sure if there is any policy about public/private hooks. Should I start names with `_`?

Reply:
Yeah, that's best, IMO. We don't want APIs added accidentally. Another option is to explicitly `del` them after use.